src/components/ModalVote.vue

```diff
@@ -91,8 +91,10 @@ async function handleSubmit() {
 }
 
 async function loadVotingValidation() {
-  if (!props.proposal.validation || props.proposal.validation.name === 'any')
-    return (isValidVoter.value = true);
+  if (!props.proposal.validation || props.proposal.validation.name === 'any') {
+    isValidVoter.value = true;
+    return;
+  }
```

Comment:
This is nice, is it possible to similar check for proposal validations too? can't find it 

Check if it is basic validation and minScore is 0

Reply:
Yes good catch, added that :)